docs/2024/index.md

```diff
@@ -52,7 +52,7 @@ More info to come here.
 | Topic(s)            | Timings                       | Meeting link                                                                                            | ICS                                  |
 | :------------------ | :---------------------------- | :------------------------------------------------------------------------------------------------------ | :----------------------------------- |
 | General Meeting     | _Thursday_ 13:30 - 14:30 UTC  | [Jitsi](https://meet.jit.si/moderated/5a655b3b6f3b4f83cddb13b93ac5408d6de48bf4ce1049f4128aa1c885478d48) | [.ics](/ics/gsoc_2024_weekly.ics)    |
-| -                   | -                             | -                                                                                                       | -                                    |
```

Comment:
Please keep the empty row after General Meeting. It is intentional to separate project specific meetings from General Meet.

Reply:
Thank you @GMishx for the change request. It's resolved now!